src/app/features/routing/routes.ts

```diff
@@ -1 +1,5 @@
-export const ROUTE_SIGN_UP = "/sign-up/";
+export const ROUTES = {
+  signUp: "/sign-up/",
+  signIn: "/sign-in/",
+  resetPassword: "/reset-password/"
+};
```

Comment:
Reason i used separate consts for each route is to allow unused variables check to detect routes which are unused in code. 
Do you have a reason to wrap these in object?